<html>
<head>
<title>TestNG:  Create user Tests</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Create user Tests</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>0/2/0</td>
</tr><tr>
<td>Started on:</td><td>Wed Aug 28 07:26:14 EDT 2019</td>
</tr>
<tr><td>Total time:</td><td>80 seconds (80285 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestCases.LoginTest.Verify_css_value_of_login_page_fields()'><b>Verify_css_value_of_login_page_fields</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_08 To verify css value of login page fields</td>
<td><div><pre>java.lang.AssertionError: expected [rgba(54, 108, 185, 1)] but found [rgba(2, 192, 245, 1)]
	at TestCases.LoginTest.Verify_css_value_of_login_page_fields(LoginTest.java:182)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1048128739", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1048128739'><pre>java.lang.AssertionError: expected [rgba(54, 108, 185, 1)] but found [rgba(2, 192, 245, 1)]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:575)
	at org.testng.Assert.assertEquals(Assert.java:585)
	at TestCases.LoginTest.Verify_css_value_of_login_page_fields(LoginTest.java:182)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>1</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_forgot_password_link()'><b>Verify_forgot_password_link</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_09 To verify forgot password link </td>
<td><div><pre>java.lang.AssertionError: expected [0] but found [1]
	at TestCases.LoginTest.Verify_forgot_password_link(LoginTest.java:219)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1250644519", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1250644519'><pre>java.lang.AssertionError: expected [0] but found [1]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:839)
	at org.testng.Assert.assertEquals(Assert.java:849)
	at TestCases.LoginTest.Verify_forgot_password_link(LoginTest.java:219)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>10</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_login_functionality_with_public_user()'><b>Verify_login_functionality_with_public_user</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_12 To verify public user login functionality with sign up via app</td>
<td><div><pre>java.lang.AssertionError: Key - &apos;appPublicUserEmail&apos; does not exist or not given a value in Sys.properties file 

Common.ConfigManager.getProperty(ConfigManager.java:44)
TestCases.LoginTest.Verify_login_functionality_with_public_user(LoginTest.java:286)
java.util.ArrayList.forEach(Unknown Source)
	at Common.ConfigManager.getProperty(ConfigManager.java:44)
	at TestCases.LoginTest.Verify_login_functionality_with_public_user(LoginTest.java:286)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 55 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1898751355", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1898751355'><pre>java.lang.AssertionError: Key - &apos;appPublicUserEmail&apos; does not exist or not given a value in Sys.properties file 

Common.ConfigManager.getProperty(ConfigManager.java:44)
TestCases.LoginTest.Verify_login_functionality_with_public_user(LoginTest.java:286)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
	at org.testng.Assert.fail(Assert.java:97)
	at Common.ConfigManager.getProperty(ConfigManager.java:44)
	at TestCases.LoginTest.Verify_login_functionality_with_public_user(LoginTest.java:286)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_login_functionality_with_valid_credentials_of_superAdmin()'><b>Verify_login_functionality_with_valid_credentials_of_superAdmin</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_11 To verify login functionality with valid credentials of super admin <br/>
<a href="#Output-1710228600" onClick='toggleBox("Output-1710228600", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1710228600" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1710228600">
Unable to verify login functionality with valid credentials of super admin 
TestCases.LoginTest.Verify_login_functionality_with_valid_credentials_of_superAdmin(LoginTest.java:274)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Unable to verify login functionality with valid credentials of super admin
	at TestCases.LoginTest.Verify_login_functionality_with_valid_credentials_of_superAdmin(LoginTest.java:275)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1710228600", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1710228600'><pre>java.lang.AssertionError: Unable to verify login functionality with valid credentials of super admin
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.LoginTest.Verify_login_functionality_with_valid_credentials_of_superAdmin(LoginTest.java:275)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>34</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_login_page_title()'><b>Verify_login_page_title</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_001 To verify login page title</td>
<td><div><pre>java.lang.AssertionError: expected [SaferWatch - Command Center] but found [Login]
	at TestCases.LoginTest.Verify_login_page_title(LoginTest.java:34)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace849776463", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace849776463'><pre>java.lang.AssertionError: expected [SaferWatch - Command Center] but found [Login]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.assertEqualsImpl(Assert.java:136)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:575)
	at org.testng.Assert.assertEquals(Assert.java:585)
	at TestCases.LoginTest.Verify_login_page_title(LoginTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_login_page_ui_fields()'><b>Verify_login_page_ui_fields</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_003 To verify login page UI fields</td>
<td><div><pre>java.lang.AssertionError: did not expect to find [true] but found [false]
	at TestCases.LoginTest.Verify_login_page_ui_fields(LoginTest.java:69)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace962287291", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace962287291'><pre>java.lang.AssertionError: did not expect to find [true] but found [false]
	at org.testng.Assert.fail(Assert.java:97)
	at org.testng.Assert.failNotEquals(Assert.java:969)
	at org.testng.Assert.assertTrue(Assert.java:43)
	at org.testng.Assert.assertTrue(Assert.java:53)
	at TestCases.LoginTest.Verify_login_page_ui_fields(LoginTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>15</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.CreateUserTest.Verify_org_user_login_functionality_with_publish_alerts_and_receive_alerts_permission()'><b>Verify_org_user_login_functionality_with_publish_alerts_and_receive_alerts_permission</b><br>Test class: TestCases.CreateUserTest<br>Test method: TC_CreateOrgUser_8 To Verify org user login functionality with 'publish alerts & receive alerts' permission <br/>
<a href="#Output-1206946689" onClick='toggleBox("Output-1206946689", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1206946689" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1206946689">
Unable To Verify org user login functionality with 'publish alerts & receive alerts' permission 
TestCases.CreateUserTest.Verify_org_user_login_functionality_with_publish_alerts_and_receive_alerts_permission(CreateUserTest.java:510)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Verify org user login functionality with &apos;publish alerts &amp; receive alerts&apos; permission
	at TestCases.CreateUserTest.Verify_org_user_login_functionality_with_publish_alerts_and_receive_alerts_permission(CreateUserTest.java:511)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1206946689", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1206946689'><pre>java.lang.AssertionError: Verify org user login functionality with &apos;publish alerts &amp; receive alerts&apos; permission
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.CreateUserTest.Verify_org_user_login_functionality_with_publish_alerts_and_receive_alerts_permission(CreateUserTest.java:511)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>40</td>
<td>TestCases.CreateUserTest@78047b92</td></tr>
<tr>
<td title='TestCases.CreateUserTest.Verify_org_user_login_functionality_with_receive_alerts_permission()'><b>Verify_org_user_login_functionality_with_receive_alerts_permission</b><br>Test class: TestCases.CreateUserTest<br>Test method: TC_CreateOrgUser_9 To Verify org user login functionality with 'receive alerts' permission <br/>
<a href="#Output-31906520" onClick='toggleBox("Output-31906520", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-31906520" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-31906520">
Unable to Verify org user login functionality with 'receive alerts' permission 
TestCases.CreateUserTest.Verify_org_user_login_functionality_with_receive_alerts_permission(CreateUserTest.java:586)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Verify org user login functionality with &apos;receive alerts&apos; permission 
	at TestCases.CreateUserTest.Verify_org_user_login_functionality_with_receive_alerts_permission(CreateUserTest.java:587)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace31906520", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace31906520'><pre>java.lang.AssertionError: Verify org user login functionality with &apos;receive alerts&apos; permission 
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.CreateUserTest.Verify_org_user_login_functionality_with_receive_alerts_permission(CreateUserTest.java:587)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>38</td>
<td>TestCases.CreateUserTest@78047b92</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_submitRequest_link()'><b>Verify_submitRequest_link</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_013 To verify submit request link and FAQ link<br/>
<a href="#Output-351970188" onClick='toggleBox("Output-351970188", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-351970188" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-351970188">
Unable to verify submit request link 
TestCases.LoginTest.Verify_submitRequest_link(LoginTest.java:349)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Unable to verify submit request link
	at TestCases.LoginTest.Verify_submitRequest_link(LoginTest.java:350)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace351970188", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace351970188'><pre>java.lang.AssertionError: Unable to verify submit request link
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.LoginTest.Verify_submitRequest_link(LoginTest.java:350)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>20</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_termsOfService_link()'><b>Verify_termsOfService_link</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_010 To verify terms of service link <br/>
<a href="#Output-1658785779" onClick='toggleBox("Output-1658785779", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1658785779" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1658785779">
Unable to verify Term of service page URL 
TestCases.LoginTest.Verify_termsOfService_link(LoginTest.java:252)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Unable to verify term of service page URL
	at TestCases.LoginTest.Verify_termsOfService_link(LoginTest.java:253)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1658785779", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1658785779'><pre>java.lang.AssertionError: Unable to verify term of service page URL
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.LoginTest.Verify_termsOfService_link(LoginTest.java:253)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>20</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_validation_message_with_empty_email_address_and_empty_password_field()'><b>Verify_validation_message_with_empty_email_address_and_empty_password_field</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_006 To verify validation message with empty 'email address' and empty 'password' field<br/>
<a href="#Output-405741210" onClick='toggleBox("Output-405741210", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-405741210" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-405741210">
Unable to verify validation message with empty 'email address' and empty 'password' field 
TestCases.LoginTest.Verify_validation_message_with_empty_email_address_and_empty_password_field(LoginTest.java:143)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Unable to verify validation message with empty &apos;email address&apos; and empty &apos;password&apos; field
	at TestCases.LoginTest.Verify_validation_message_with_empty_email_address_and_empty_password_field(LoginTest.java:144)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace405741210", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace405741210'><pre>java.lang.AssertionError: Unable to verify validation message with empty &apos;email address&apos; and empty &apos;password&apos; field
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.LoginTest.Verify_validation_message_with_empty_email_address_and_empty_password_field(LoginTest.java:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>30</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_validation_message_with_invalid_email_address_and_invalid_password_field()'><b>Verify_validation_message_with_invalid_email_address_and_invalid_password_field</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_007 To verify validation message with invalid 'Email address' and invalid 'password'<br/>
<a href="#Output-1478334108" onClick='toggleBox("Output-1478334108", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1478334108" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1478334108">
Unable to verify validation message with invalid 'email address' and invalid 'password' field 
TestCases.LoginTest.Verify_validation_message_with_invalid_email_address_and_invalid_password_field(LoginTest.java:167)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Unable to verify validation message with invalid &apos;email address&apos; and invalid &apos;password&apos; field
	at TestCases.LoginTest.Verify_validation_message_with_invalid_email_address_and_invalid_password_field(LoginTest.java:168)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1478334108", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1478334108'><pre>java.lang.AssertionError: Unable to verify validation message with invalid &apos;email address&apos; and invalid &apos;password&apos; field
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.LoginTest.Verify_validation_message_with_invalid_email_address_and_invalid_password_field(LoginTest.java:168)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>25</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_validation_message_with_valid_emailAddress_and_empty_password_field()'><b>Verify_validation_message_with_valid_emailAddress_and_empty_password_field</b><br>Test class: TestCases.LoginTest<br>Test method:  TC_Login_004  To verify validation message with valid 'email address' and empty 'password' field<br/>
<a href="#Output-231725600" onClick='toggleBox("Output-231725600", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-231725600" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-231725600">
Unable to verify validation message with valid 'email address' and empty 'password' field 
TestCases.LoginTest.Verify_validation_message_with_valid_emailAddress_and_empty_password_field(LoginTest.java:100)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Unable to verify validation message with valid &apos;email address&apos; and empty &apos;password&apos; field
	at TestCases.LoginTest.Verify_validation_message_with_valid_emailAddress_and_empty_password_field(LoginTest.java:101)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace231725600", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace231725600'><pre>java.lang.AssertionError: Unable to verify validation message with valid &apos;email address&apos; and empty &apos;password&apos; field
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.LoginTest.Verify_validation_message_with_valid_emailAddress_and_empty_password_field(LoginTest.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>27</td>
<td>TestCases.LoginTest@3b938003</td></tr>
<tr>
<td title='TestCases.LoginTest.Verify_validation_message_with_valid_password_and_empty_emailAddress_field()'><b>Verify_validation_message_with_valid_password_and_empty_emailAddress_field</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_005 To verify validation message with valid 'Password' and empty 'email address' field<br/>
<a href="#Output-181078353" onClick='toggleBox("Output-181078353", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-181078353" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-181078353">
Unable to verify validation message with valid 'password' and empty 'email address' field 
TestCases.LoginTest.Verify_validation_message_with_valid_password_and_empty_emailAddress_field(LoginTest.java:120)
sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
java.lang.reflect.Method.invoke(Unknown Source)
org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
java.util.ArrayList.forEach(Unknown Source)
org.testng.TestRunner.privateRun(TestRunner.java:770)
org.testng.TestRunner.run(TestRunner.java:591)
org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
org.testng.SuiteRunner.run(SuiteRunner.java:304)
org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
org.testng.TestNG.runSuites(TestNG.java:1032)
org.testng.TestNG.run(TestNG.java:1000)
org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: Unable to verify validation message with valid &apos;password&apos; and empty &apos;email address&apos; field
	at TestCases.LoginTest.Verify_validation_message_with_valid_password_and_empty_emailAddress_field(LoginTest.java:121)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace181078353", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace181078353'><pre>java.lang.AssertionError: Unable to verify validation message with valid &apos;password&apos; and empty &apos;email address&apos; field
	at org.testng.Assert.fail(Assert.java:97)
	at TestCases.LoginTest.Verify_validation_message_with_valid_password_and_empty_emailAddress_field(LoginTest.java:121)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(Unknown Source)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>30</td>
<td>TestCases.LoginTest@3b938003</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestCases.LoginTest.Verify_login_page_url()'><b>Verify_login_page_url</b><br>Test class: TestCases.LoginTest<br>Test method: TC_Login_002 To verify login page url</td>
<td></td>
<td>3</td>
<td>TestCases.LoginTest@3b938003</td></tr>
</table><p>
</body>
</html>